File: Estruturas de Repetição/Laços_de_Repetição.py
print("\nAdivinhe o número com 'while' em Python!")
numero_secreto = 7
numero_digitado = 0

while numero_digitado != numero_secreto:
        numero_digitado = int(input("Digite um número: "))
        if numero_digitado != numero_secreto:
            print ("Número incorreto. Tente Novamente.")

print("Parabéns! Você acertou!")

File: Estruturas de Repetição/Laços_de_Repetição.cpp
#include <iostream>

int main() {
    std::cout << "\nAdivinhe o número com 'while' em C++" << std::endl;
    int numero_secreto = 7;
    int numero_digitado = 0;
    
    while (numero_digitado != numero_secreto) {
        std::cout << "Digite um número ";
        std::cin >> numero_digitado;
        if (numero_digitado != numero_secreto) {
            std::cout << "Número incorreto. Tente novamente." << std::endl;
        }
    }

    std::cout << "Parabéns! Você acertou!" << std::endl;
    return 0;
}